src/views/Nft/market/Profile/components/EditProfileModal/ChangeProfilePicView.tsx

```diff
@@ -61,13 +61,13 @@ const ChangeProfilePicPage: React.FC<ChangeProfilePicPageProps> = ({ onDismiss,
         return callWithGasPrice(profileContract, 'updateProfile', [selectedNft.collectionAddress, selectedNft.tokenId])
       },
       onSuccess: async ({ receipt }) => {
-        // Re-fetch profile
-        await dispatch(fetchProfile(account))
+        onDismiss()
```

Comment:
We should dismiss the modal first from ui otherwise user would see skeleton in this version or warning in current deployed version (since nfts will first return empty array when retriggered)